### PR TITLE
feat(linter): Added fixer for duplicate prefix in valid title jest rule

### PR DESCRIPTION
**Pull request by tapanprakasht**
Supported fixer for duplicate prefix in jest title rule.

### CODE REVIEW

File: crates/oxc_linter/src/rules/jest/valid_title.rs [hunk 1]
```diff
@@ -312,9 +312,12 @@ fn validate_title(
         return;
     };
 
-    // TODO: support fixer
     if first_word == un_prefixed_name {
-        Message::DuplicatePrefix.diagnostic(ctx, span);
+        let (error, help) = Message::DuplicatePrefix.detail();
+        ctx.diagnostic_with_fix(valid_title_diagnostic(error, help, span), |fixer| {
+            let replaced_title = title[first_word.len()..].trim().to_string();
```
Comment (DonIsaac) [inline crates/oxc_linter/src/rules/jest/valid_title.rs:318]: I think to_string is unnecessary

File: crates/oxc_linter/src/rules/jest/valid_title.rs [hunk 2]
```diff
@@ -312,9 +312,12 @@ fn validate_title(
...
+            fixer.replace(Span::new(span.start + 1, span.end - 1), replaced_title)
```
Comment (DonIsaac) [inline crates/oxc_linter/src/rules/jest/valid_title.rs:319]: Consider shrink_left and shrink_right

Comment (tapanprakasht) [reply crates/oxc_linter/src/rules/jest/valid_title.rs]: Done. Moved to shrink methods.

File: crates/oxc_linter/src/rules/jest/valid_title.rs (hunk 1 above)
Comment (tapanprakasht) [reply crates/oxc_linter/src/rules/jest/valid_title.rs:318]: Tried removing the to_string(). Getting lifetime issues.

**Pull request merged**